packages/api-reference/src/components/Content/Schema/SchemaProperty.vue

```diff
@@ -314,7 +314,7 @@ const displayPropertyHeading = (
           typeof optimizedValue.items === 'object' &&
           discriminator in optimizedValue.items &&
           Array.isArray(optimizedValue.items[discriminator]) &&
-          level < 3
+          level < 5
```

Comment:
any cons in passing directly optimizedValue here?

Reply:
Idk, give it a try!